app/posts/register-trainee-teachers/2021-05-08-courses-with-multiple-subjects.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Courses with multiple subjects
+description: Adding support for a second and third subject on a course
+date: 2021-05-08
+---
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+{% from "figure/macro.njk" import appFigure with context %}
+
+When trainees are added to DTTP, up to three subjects can be recorded for their course - these indicate what they were specifically trained to teach in. Register previously only had a single subject, so we’ve added the ability to add a second and third subject.
+
+
+{{ appFigure({
+  title: "Previous subject field",
+  image: {
+    file: "subject-existing.png",
+    alt: "Our previous subject field."
+  }
+}) }}
+
+{{ appFigure({
+  title: "Revised subject fields",
+  image: {
+    file: "subject-multiple.png",
+    alt: "Our revised subject input - a details component reveals an autocomplete for second and third subject"
+  }
+}) }}
+
+## Implementation details
+
+The first subject is the most important subject - it gets used to help determine bursary eligibility and allocations. The others are optional and give more detail for what the trainee’s course covered.
+
+The second and third subjects are collapsed by default because the majority of trainees only have a single subject.
+
+The DfE Publish service supports courses with up to two subjects, so when we import trainees who are on a Publish course we’ll now be able to populate both subjects.
```

Comment:
This might be a bit unclear to a newcomer. Maybe split it up, eg: '...up to 2 subjects. We'll now be able to populate both subject fields when we import trainees who are on courses listed on Publish.'